Statut HSTrójmiasto.tex

```diff
@@ -0,0 +1,254 @@
+	        \item podejmowanie uchwał o zarządzaniu majątkiem Stowarzyszenia,
+	        \item reprezentowanie Stowarzyszenia na zewnątrz,
+	        \item zwoływanie Walnego Zebrania Członków,
+	        \item przyjmowanie i skreślanie członków zwyczajnych,
+	        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
+	        \item sporządzanie rocznego sprawozdania finansowego,
+	        \item zwalnianie w uzasadnionych przypadkach Członków Zwyczajnych z obowiązku płacenia składek w całości bądź w części.
+	      \end{enumerate}
+	    \item Posiedzenia Zarządu odbywają się w miarę potrzeb, nie rzadziej jednak niż raz na rok.
+	    \item Posiedzenia Zarządu zwołuje:
+	      \begin{enumerate}
+	        \item dwóch członków zarządu działających łącznie,
+	        \item Komisja Rewizyjna.
+	      \end{enumerate}
+	    \item Członek Zarządu może zrezygnować z pełnienia funkcji poprzez złożenie oświadczenia na Walnym Zebraniu Członków.
+	\end{enumerate}
+    \begin{enumerate}
+	    \item Komisja Rewizyjna powoływana jest do sprawowania kontroli nad działalnością Stowarzyszenia.
+	    \item Komisja Rewizyjna składa się z 3 równorzędnych członków.
+	    \item Do kompetencji Komisji Rewizyjnej należy:
+	      \begin{enumerate}
+	        \item kontrolowanie działalności Zarządu,
+	        \item składanie sprawozdań z kontroli na Walnym Zebraniu Członków,
+	        \item składanie wniosków o udzielenie absolutorium dla Zarządu Stowarzyszenia,
+	        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
+	        \item zatwierdzanie rocznego sprawozdania finansowego.
+	      \end{enumerate}
+	    \item Członek Komisji Rewizyjnej może zrezygnować z pełnienia funkcji poprzez złożenie oświadczenia na Walnym Zebraniu Członków.
+	\end{enumerate}
+\section{MAJĄTEK I FUNDUSZE}
+  \begin{enumerate}
+    \item Majątek Stowarzyszenia składa się z:
+      \begin{enumerate}
+        \item składek członkowskich,
+        \item subwencji, darowizn, spadków i zapisów,
+        \item wpływów z odpłatnej działalności statutowej,
+        \item wpływów z ofiarności publicznej,
+        \item dochodów z majątku, odsetek oraz kapitału,
+        \item dotacji,
+        \item wpływów z loterii, aukcji i sponsoringu,
+        \item zbiórek publicznych.
+      \end{enumerate}
+    \item Stowarzyszenie prowadzi gospodarkę finansową zgodnie z obowiązującymi przepisami.
+    \item Organem kompetentnym w zakresie zarządzania majątkiem Stowarzyszenia jest Zarząd.
+    \item Zarząd zobowiązany jest dołożyć wszelkich starań w celu utrzymania zapasu środków na kontach Stowarzyszenia wystarczającego na pokrycie stałych zobowiązań Stowarzyszenia przez okres co najmniej dwóch miesięcy.
+    \item Do zawierania umów, zwłaszcza w sprawach majątkowych i niemajątkowych, udzielania pełnomocnictwa i składania innych oświadczeń woli, w szczególności w sprawach majątkowych upoważnieni są:
```

Comment:
Jak to wygląda jak kupuję za hajs HSu dajmy na to switcha za 100 cebulinów? Papir podpisany przez 2 osoby?

Reply:
tak, dokladnie. Dlatego sugeruje ze jeden czlonek zarzadu moze podejmowac decyzje bez podpisu drugiego do kwoty 5k jesli nie jest to dlugoterminowe zobowiazanie.